src/Abstractions/NexusMods.Abstractions.Loadouts.Synchronizers/ALoadoutSynchronizer.cs

```diff
@@ -85,7 +85,7 @@ private void CleanDirectories(IEnumerable<GamePath> toDelete, DiskState newState
         var seenDirectories = new HashSet<GamePath>();
         var directoriesToDelete = new HashSet<GamePath>();
 
-        var newStatePaths = newState.SelectMany(e => ((GamePath)e.Path).GetAllParents()).ToHashSet();
+        var newStatePaths = newState.SelectMany(e => ((GamePath)e.Path).GetAllParents().Skip(1)).ToHashSet();
```

Comment:
Why are we skipping 1?

Reply:
The first one is the full file path, which we don't need since we only care about the folders.

Reply:
You should put a comment in there.